variables.tf

```diff
@@ -75,3 +75,12 @@ variable "ec2_ingress_sg_id" {
   type        = list(string)
   default     = []
 }
+
+//----------------------------------------------------------------------
+// Networking Variables
```

Comment:
I would change this from Networking Variables since there's already a heading for this (line 18). A more appropriate heading here would be something like "ECS Cluster Variables".

Reply:
Updated